### ISSUE TITLE
QQ video cannot download

### ISSUE THREAD

**Issue by scegg** [soimort/you-get#2020]
Please make sure these boxes are checked before submitting your issue – thank you!

- [ ] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [ ] Your `you-get` is up-to-date.
- [ ] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [ ] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [ ] The issue (or question) is really about `you-get`, not about some other code or project.

Run the command with the `--debug` option, and paste the full output inside the fences:

```
>you-get --debug  https://v.qq.com/x/cover/e4th0v7bfha4rrf/g0506r4ca9a.html
[DEBUG] get_html: http://v.qq.com/x/cover/e4th0v7bfha4rrf/g0506r4ca9a.html
[DEBUG] get_response: http://v.qq.com/x/cover/e4th0v7bfha4rrf/g0506r4ca9a.html
[DEBUG] get_html: http://vv.video.qq.com/getinfo?otype=json&appver=3%2E2%2E19%2E333&platform=11&defnpayver=1&vid=g0506r4ca9a
[DEBUG] get_response: http://vv.video.qq.com/getinfo?otype=json&appver=3%2E2%2E19%2E333&platform=11&defnpayver=1&vid=g0506r4ca9a
[DEBUG] get_html: http://vv.video.qq.com/getkey?otype=json&platform=11&format=10209&vid=g0506r4ca9a&filename=g0506r4ca9a.p209.1.mp4
[DEBUG] get_response: http://vv.video.qq.com/getkey?otype=json&platform=11&format=10209&vid=g0506r4ca9a&filename=g0506r4ca9a.p209.1.mp4
[DEBUG] url_info: http://113.105.141.26/vkp.tc.qq.com/g0506r4ca9a.p209.1.mp4?vkey=7D1F74EFB38DBF29828144F9ECAA4D26493357D1EA77080B727BD2B35FEC8DC6891AE7A1FBE3ADA99EE4F059CDED4E69066B1BF8A20EBBEE52A5BE4B3A382D8817CE18376D6FFAB0FBDE824651F186A9A5EAC77F110C38C3E34FFE9FBD62E73F
you-get: version 0.4.715, a tiny downloader that scrapes the web.
you-get: ['https://v.qq.com/x/cover/e4th0v7bfha4rrf/g0506r4ca9a.html']
Traceback (most recent call last):
  File "<string>", line 11, in <module>
  File "src\you_get\__main__.py", line 92, in main
  File "src\you_get\common.py", line 1429, in main
  File "src\you_get\common.py", line 1342, in script_main
  File "src\you_get\common.py", line 1158, in download_main
  File "src\you_get\common.py", line 1422, in any_download
  File "src\you_get\extractors\qq.py", line 158, in qq_download
  File "src\you_get\extractors\qq.py", line 57, in qq_download_by_vid
UnboundLocalError: local variable 'ext' referenced before assignment
you-get returned -1
```

If there's anything else you would like to say (e.g. in case your issue is not about downloading a specific video; it might as well be a general discussion or proposal for a new feature), fill in the box below; otherwise, you may want to post an emoji or meme instead:

> [WRITE SOMETHING]
> [OR HAVE SOME :icecream:!]

汉语翻译最终日期：2016年02月26日

在提交前，请确保您已经检查了以下内容!

- [ ] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [ ] 您的`you-get`为最新版.
- [ ] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [ ] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [ ] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

```
[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]
> [舔 :icecream:!]

**Comment by scegg** [comment]
could be merged into [#2054](https://github.com/soimort/you-get/issues/2054).
reason: same file discussed and workaround provided.

**Comment by rosynirvana** [comment]
```
$ ./you-get -i 'https://v.qq.com/x/cover/e4th0v7bfha4rrf/g0506r4ca9a.html'
Site:       QQ.com
Title:      【科技相对论】互联网传奇企业——亚马逊的成功之道
Type:       MPEG-4 video (video/mp4)
Size:       183.12 MiB (192016269 Bytes)
```
